### PR TITLE
Comment list created and relates to specified article, basic styling

### PR DESCRIPTION
**Pull request by MHWatchy**
Able to view comments related to an individual article, uses a temporary route of /articles/:article_id/comments as individual articles, where the comments would be displayed, are in another branch

### CODE REVIEW

**Review by popatre — CHANGES_REQUESTED** [review]
- Nice comment component to stop the main component getting too large
- Well remembered to add the loading

- [ ] You should add some feedback for when an article has no comments

Also, im not quite sure how im supposed to navigate there, but presume this is on another branch?

File: src/components/Article.jsx [hunk 1]
```diff
@@ -17,22 +18,25 @@ const Article = () => {
...
-      />
-      <p>{article.body}</p>
-      <h3>{article.votes} votes</h3>
-    </section>
+    <>
+      <section>
+        <h3>{article.topic}</h3>
+        <h1>{article.title}</h1>
+        <h2>by {article.author}</h2>
+        <h3>{article.created_at}</h3>
```
Comment (popatre) [inline src/components/Article.jsx]: You shouldnt use h1 and h3 etc for sizing. They go in order i.e. h1 for main title
h2 for sub titles - h3 for a sub titles inside the other sub. 

You can change the sizes via css, but should never use them to get different sizes

**Review by popatre — APPROVED** [review]
Looks good, but please see my comment regarding h1,2,3 etc

**Comment by nc-modou** [comment]
Good response to feedback, comment cards lookin good - merge away!

**Pull request merged**